kubernetes/ingress.go

```diff
@@ -71,6 +71,7 @@ const (
 	ingressLoadBalancerTypeAnnotation = "zalando.org/aws-load-balancer-type"
 	ingressHTTP2Annotation            = "zalando.org/aws-load-balancer-http2"
 	ingressClassAnnotation            = "kubernetes.io/ingress.class"
+	clusterLocalDomain                = ".cluster.local"
```

Comment:
Let's make this a flag because users might have a different cluster domain depending on their setup.